src/Compilers/Test/Core/Compilation/OperationTreeVerifier.cs

```diff
@@ -469,6 +469,13 @@ public override void VisitUsingDeclaration(IUsingDeclarationOperation operation)
             LogCommonPropertiesAndNewLine(operation);
 
             Visit(operation.DeclarationGroup, "DeclarationGroup");
+
+            var disposeMethod = ((UsingDeclarationOperation)operation).DisposeMethod;
```

Comment:
>disposeMethod [](start = 16, length = 13)

I think it would be good to include this symbol into the output when it is not null.